geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ingest/IngestCommand.scala

```diff
@@ -169,4 +177,10 @@ trait IngestParams extends OptionalTypeNameParam with OptionalFeatureSpecParam
     with OptionalConverterConfigParam with OptionalInputFormatParam with DistributedRunParam {
   @Parameter(names = Array("-t", "--threads"), description = "Number of threads if using local ingest")
   var threads: Integer = 1
+
+  @Parameter(names = Array("--split-max-size"), description = "Maximum size of a split in bytes")
+  var maxSplitSize: Integer = -1
+
+  @Parameter(names = Array("--src-list"), description = "Input files are text files with lists of files, one per line, to ingest.")
```

Comment:
should we make this arity 1?

Reply:
Don't think so, that'd make it true by default.

Reply:
i thought you could make it false by default and then pass `--src-list` to make it true

Reply:
oh nvm, i keep forgetting it will make it arity 1 by default